recipes/ncbi-cxx-toolkit-public/26.0/test_package/CMakeLists.txt

```
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.7)
```

```suggestion
cmake_minimum_required(VERSION 3.8)
```

CMake 3.7 doesn't understand  CXX_STANDARD 17

---

indeed